vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -439,7 +438,6 @@ type internal FSharpLanguageService(package : FSharpPackage) =
             | WorkspaceChangeKind.ProjectAdded     -> this.OnProjectAdded(args.ProjectId)
             | WorkspaceChangeKind.ProjectReloaded  -> this.OnProjectReloaded(args.ProjectId)
             | WorkspaceChangeKind.DocumentAdded    -> this.OnDocumentAdded(args.ProjectId, args.DocumentId)
-            | WorkspaceChangeKind.DocumentChanged  -> this.OnDocumentChanged(args.ProjectId, args.DocumentId)
```

Comment:
Strange that `DocumentChanged` was in this list twice already.  Also - consider whether `DocumentInfoChanged` should result in a call.

Reply:
Thinking about this, once we move away from handlecommandlinargs, we will need to listen to DocumentInfoChanged most likely.

Reply:
We didn't see any DocumentInfoChanged  events - when do they arise?

Reply:
For example when the filename changes, etc:
```C#
        /// <summary>
        /// The document in the current solution had is info changed; name, folders, filepath
        /// </summary>
        DocumentInfoChanged = 17,
```